email_app/package.json

```diff
@@ -129,8 +129,8 @@
     "urlsafe-base64": "^1.0.0"
   },
   "devEngines": {
-    "node": "4.x || 5.x || 6.x || 7.x",
-    "npm": "2.x || 3.x || 4.x"
+    "node": "4.x || 5.x || 6.x || 8.x",
```

Comment:
By doing this change we are saying we don't support v7.x but we actually do, I think you should be adding v8.x to the list rather than replacing v7.x with it (I guess the same comment goes for the npm versions).
Also, I just gave it a try to install with v4.x and v5.x, it fails in both cases with an error about an incompatible module, I therefore assume we are not supporting v4.x and v5.x anymore, thus we should remove them from the list:
```
error mime@2.0.3: The engine "node" is incompatible with this module. Expected version ">=6.0.0".
error Found incompatible module
```
We could alternatively restrict it to just v8.x if we wanted to, I would be ok with that too since that's what we are saying in the README.